crates/common/tedge_config/src/tedge_config_cli/new.rs

```diff
@@ -373,7 +373,7 @@ define_tedge_config! {
 
         /// Set of MQTT topics the Cumulocity mapper should subscribe to
         #[tedge_config(example = "tedge/alarms/#,tedge/measurements/+")]
-        #[tedge_config(default(value = "tedge/measurements,tedge/measurements/+,tedge/alarms/+/+,tedge/alarms/+/+/+,tedge/events/+,tedge/events/+/+,tedge/health/+,tedge/health/+/+"))]
+        #[tedge_config(default(value = "tedge/measurements,tedge/measurements/+,tedge/alarms/+/+,tedge/alarms/+/+/+,tedge/events/+,tedge/events/+/+,tedge/health/+,tedge/health/+/+,te/device/+///m/,te/device/+///m/+,te/device/+///e/,te/device/+///e/+,te/device/+///a/,te/device/+///a/+"))]
```

Comment:
fixed

Reply:
This PR only addresses the `telemetry` part for the devices not for the `services`.

Reply:
Please implement for services as well as it is 99% the same logic. I'm ok if we ignore the auto-registration for services, but at least publishing telemetry data for everything makes more sense as the purpose of the PR is to "New topics for telemetry", so extending it to also support services would be useful as it will be touching the same function (so we should prevent double work here).

Reply:
sure